pkg/operator/ceph/file/mds/mds.go

```diff
@@ -126,6 +127,16 @@ func (c *Cluster) Start() error {
 	// Always create double the number of metadata servers to have standby mdses available
 	replicas := c.fs.Spec.MetadataServer.ActiveCount * 2
 
+	mdsToSkipReconcile, err := c.getMDSsToSkipReconcile()
+	if err != nil {
+		return errors.Wrap(err, "failed to check for mds to skip reconcile")
+	}
+
+	if mdsToSkipReconcile.Len() > 0 {
+		logger.Warningf("skipping mgr reconcile since mdss are labeled with %s: %v", cephv1.SkipReconcileLabelKey, mdsToSkipReconcile.List())
```

Comment:
```suggestion
		logger.Warningf("skipping mds reconcile since mds daemons are labeled with %q: %v", cephv1.SkipReconcileLabelKey, mdsToSkipReconcile.List())
```